### PR TITLE
Fix possible redefinition of "-ccbin" in `cupy.fft._callback`

### PR DESCRIPTION
**Pull request by leofang**
refs:
- https://jenkins.preferred.jp/job/chainer/job/daily_master_cupy-shuffled/ID=18,label=mn1-p100/383/console
- `environment.sh:        export NVCC="${NVCC} --compiler-bindir gcc-6"`

cc: @emcastillo

### CODE REVIEW

**Comment by leofang** [comment]
Jenkins, test this please

File: cupy/fft/_callback.pyx [hunk 1]
```diff
@@ -96,6 +75,38 @@ cdef class _ThreadLocal:
...
+    cdef int i
+
+    global _nvcc
+    if _nvcc == []:
+        nvcc = get_nvcc_path()
+        if nvcc is not None:
+            _nvcc = nvcc.split(' ')
+            # if the host compiler is set in NVCC, we force it to align with
+            # the one reported by sysconfig
+            for i, f in enumerate(_nvcc):
```
Comment (emcastillo) [inline cupy/fft/_callback.pyx:100]: why don't we use the one that is provided in nvcc?

Comment (leofang) [reply cupy/fft/_callback.pyx:100]: My original thought was I wanna use the compiler Python says it'd use for building extensions, but now think it also makes sense to do the other way around. Will change.

Comment (leofang) [reply cupy/fft/_callback.pyx:101]: Changed in 30a620d. @emcastillo PTAL.

**Comment by emcastillo** [comment]
Jenkins, test this please

**Review by emcastillo — APPROVED** [review]

**Pull request merged**